GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -701,10 +705,17 @@ private synchronized void writeRow(Row row, SurveyInstanceDto dto,
                         String columnLocale = useQID ? "en" : locale;
 
                         if (QuestionType.GEO == q.getType()) {
-                            createCell(row, offset++,
-                                    (useQID ? questionId + "_" : q.getKeyId().toString() + "|")
-                                            + LAT_LABEL.get(columnLocale),
-                                    headerStyle);
+                            if (useQuestionId) {
+                                createCell(row, offset++,
+                                        (useQID ? questionId + "_" : q.getText() + " - ")
+                                                + LAT_LABEL.get(columnLocale),
+                                        headerStyle);
+                            } else {
+                                createCell(row, offset++,
+                                        (useQID ? questionId + "_" : q.getKeyId() + "|")
+                                                + LAT_LABEL.get(columnLocale),
+                                        headerStyle);
```

Comment:
In this second branch of the if statement the first option of the ternary `useQID ? questionId + "_" :` will never be executed no? Maybe you can just output the `keyId` with the pipe.